shell/platform/android/io/flutter/embedding/engine/FlutterEngine.java

```diff
@@ -205,6 +210,7 @@ public FlutterEngine(
     flutterLoader.ensureInitializationComplete(context, dartVmArgs);
 
     flutterJNI.addEngineLifecycleListener(engineLifecycleListener);
+    flutterJni.setPlatformViewsController(platformViewsController);
```

Comment:
my mistake: `flutterJNI`

Reply:
my bad, I should have caught that- should be fixed now